metis-enrichment/metis-enrichment-client/src/main/java/eu/europeana/enrichment/rest/client/EnrichmentWorker.java

```diff
@@ -0,0 +1,362 @@
+package eu.europeana.enrichment.rest.client;
+
+import java.io.UnsupportedEncodingException;
+import java.util.ArrayList;
+import java.util.Iterator;
+import java.util.List;
+import java.util.Set;
+import org.jibx.runtime.JiBXException;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import eu.europeana.corelib.definitions.jibx.IsPartOf;
+import eu.europeana.corelib.definitions.jibx.PlaceType;
+import eu.europeana.corelib.definitions.jibx.RDF;
+import eu.europeana.corelib.definitions.jibx.ResourceOrLiteralType;
+import eu.europeana.enrichment.api.exceptions.UnknownException;
+import eu.europeana.enrichment.api.external.model.EnrichmentBase;
+import eu.europeana.enrichment.api.external.model.EnrichmentResultList;
+import eu.europeana.enrichment.utils.EnrichmentUtils;
+import eu.europeana.enrichment.utils.InputValue;
+import eu.europeana.metis.dereference.DereferenceUtils;
+import eu.europeana.metis.dereference.client.DereferenceClient;
+
+/*
+ * To be integrated into eCloud topology. This code serves as an example of enrichment through use of the
+ * enrichment and dereferencing clients and utils. 
+ * 
+ */
+public class EnrichmentWorker {
```

Comment:
Maybe we can add more comments to this class? We are trying to tell them how to use the code that we supply. Maybe it would be great for them to have some explanation on what the calls do.

Reply:
Some extra comments have been added but overall the flow of the logic (steps), together with the related usage of the utils/clients is quite clear.